tests/components/aosmith/test_config_flow.py

```diff
@@ -0,0 +1,125 @@
+"""Test the A. O. Smith config flow."""
+from unittest.mock import AsyncMock, patch
+
+from py_aosmith import AOSmithInvalidCredentialsException
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.aosmith.const import DOMAIN
+from homeassistant.const import CONF_EMAIL
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+from tests.components.aosmith.conftest import FIXTURE_USER_INPUT
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Comment:
Don't think you need this one (I also removed this from the config flow scaffold recently)

Reply:
Removed.